frontend/src/pages/spells/[slug]/index.js

```diff
@@ -0,0 +1,69 @@
+import SpellDetails from '../../../components/pages/spells/[slug]/SpellDetails'
+import SpellMeta from '../../../components/pages/spells/[slug]/SpellMeta'
+import { getSpells, getSpellsBySlug } from '../../../lib/load_spells';
+
+const Spells = ({ data, links }) => {
+  const { attributes } = data;
+
+  return (
+    <>
+      <SpellMeta attributes={attributes} />
+      <SpellDetails attributes={attributes} apiLink={links?.self} />
+    </>
+  )
+}
+
+export async function getStaticPaths() {
+  const defaultSlugs = [
+    "accio",
+    "alohomora",
+    "avada-kedavra",
+    "expecto-patronum",
+    "expelliarmus",
+    "imperio",
+    "lumos",
+    "obliviate",
+    "stupefy",
+    "wingardium-leviosa",
+  ]
```

Comment:
This won't work as you must use the slugs from the api. Just look up the slug from the api for each of those and replace them with the results. 

example for avada kedavra: https://api.potterdb.com/v1/spells?filter[incantation_cont_any]=avada
=> slug: `killing-curse`